foundation_cms/static/scss/blocks/themes/default/featured_container_block.scss

```diff
@@ -0,0 +1,73 @@
+.featured-container {
+  $container-padding: 8rem;
+  $sticky-top-offset: $container-padding + 3rem;
+
+  display: flex;
+  flex-direction: column;
+  padding: 3rem 0;
+  gap: $global-padding;
+
+  @include breakpoint(medium up) {
+    flex-direction: row;
+    gap: $global-padding * 2;
+    padding: $container-padding 0;
+  }
+
+  &--middle {
+    @include breakpoint(medium up) {
+      align-items: center;
+    }
+  }
+
+  &--bottom {
+    @include breakpoint(medium up) {
+      align-items: flex-end;
+    }
+  }
+
+  &--top {
+    @include breakpoint(medium up) {
+      align-items: flex-start;
+    }
+  }
+
+  &--sticky-left {
+    @include breakpoint(medium up) {
+      .featured-container__column--left {
+        position: sticky;
+        top: $sticky-top-offset;
+      }
+    }
+  }
+
+  &--sticky-right {
+    @include breakpoint(medium up) {
+      .featured-container__column--right {
+        position: sticky;
+        top: $sticky-top-offset;
+      }
+    }
+  }
+
+  &__column {
+    flex: 1;
+
+    .grid-container {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  .media-block {
+    margin: 0;
+  }
+
+  .media-block__image,
+  .media-block__video {
+    border-bottom-left-radius: 0;
+  }
+
+  :is(h2, h3, h4, h5, h6, p):first-child {
```

Comment:
Nice syntax! We'll also need to target `ul` and `ol` as well, or perhaps just `:first-child` regardless of its tag type.